vignettes/ParametricCopulaFamilies.Rmd

```diff
@@ -0,0 +1,163 @@
+---
+title: "Parametric Copula Families and Their Functions"
+params:
+  do_calc: FALSE
+output:
+  bookdown::html_vignette2:
+    toc: true
+author: "Alan Kuchinsky, Elif F. Acar, Martin Lysy"
+date: 2024-03-26
+bibliography: copulaReferences.bib
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Parametric Copula Families and Their Functions}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+header-includes:
```

Comment:
Preferable not to have the `header-includes`:

- `\usepackage{xcolor}` is ignored for vignette output (nor is it actually needed here).
- \setmathfont{STIXTwoMath-Regular.otf} seems to only be used for `\mathcal{}` of lower case letters.  Probably best to avoid fancy fonts.  You can just write e.g., `log c(u, v | theta) = ...` thus avoiding the `\mathcal{}` notation altogether.